FastSurferCNN/data_loader/augmentation.py

```diff
@@ -107,7 +107,7 @@ def __init__(self, output_size, pos='top_left'):
         :param output_size:
         :param pos: position to put the input
         """
-        if isinstance(output_size, float):
+        if isinstance(output_size, int):
```

Comment:
probably should be numbers.Number, and then `output_size = (int(output_size),) * 2`
same for ZeroPad2DTest

Reply:
Done (see below).